KillTeam/Views/ListEquipesPage.xaml.cs

```diff
@@ -13,7 +14,9 @@ public ListEquipesPage()
             InitializeComponent();
             On<Xamarin.Forms.PlatformConfiguration.iOS>().SetUseSafeArea(true);
 
-            var vm = new ListEquipesViewModel(ToolbarItems);
+            var deleteTeamCommandHandler = new DeleteTeamCommandHandler();

```

Comment:
Should we use the same handler object for the `EquipePage` and the `ListEquipesPage`? Or is there no value in re-use here for a cost in refactoring to get there?

Reply:
Cost to benefit isn't there yet, however making KTContext into something that can be injected into this handler (and future handlers) would make the refactor worth it.

This is a step along the road to having Page -> ViewModel -> Query/Command -> Model.